Makefile

```diff
@@ -320,7 +320,7 @@ rbac-manifests:
 
 .PHONY: manager-manifests
 manager-manifests:
-	make -C operator docker-build
+	make -C operator docker-build docker-push
```

Comment:
I think we always want to split build and deploy to avoid unintended push.

Reply:
+1. I think the naming should probably more clear. Its not intuitive from the outside if "manager-manifests" is a build command or a push/deploy command. I think its okay to have a deploy/push command depend on a build command.